docs/user/guides/build/webhooks.rst

```diff
@@ -80,7 +61,7 @@ you will see the server response, the webhook request, and the payload.
    Activity of a webhook
 
 Custom payload examples
-~~~~~~~~~~~~~~~~~~~~~~~
+-----------------------
```

Comment:
I think we shouldn't change these here. `~~~~` are used as subsections of `Build status webhooks` and I think it's correct to be subsections.

Reply:
I reverted it :+1: 

But essentially, it's because I wanted to go towards a step-wise how-to (and not nested sections like we have in reference or explanation). Unfortunately, this how-to is very far from such a result :)

In general, how-tos are better like....

```
Do this
-------

lorem ipsum

Then do this
------------

lorem ipsum
```

Reply:
The following two were not updated and they should because of the same reason.

I understand what you want to achieve with the sections, but this case doesn't work like that since they are subsections and not steps the user has to follow.